### PR TITLE
Update shared-how-license-microsoft.md

### PR DESCRIPTION
**Pull request by SteveHallUKCLoud**
Updated to make it three options to license - added the option for customers to use their own SPLA.

### CODE REVIEW

File: articles/shared/shared-how-license-microsoft.md [hunk 1]
```diff
@@ -24,12 +24,14 @@ toc_mdlink: shared-how-license-microsoft.md
...
 This article provides information on how you can apply Microsoft licensing on our platform and the options available to you.
 
-UKCloud offers two ways to license Microsoft software for use on our compute platform, providing you with the flexibility you need to deliver your application workloads in line with Microsoft requirements and obligations:
+UKCloud offers three ways to license Microsoft software for use on our compute platform, providing you with the flexibility you need to deliver your application workloads in line with Microsoft requirements and obligations:
 
 - UKCloud can purchase licences monthly for you
 
 - You can submit a [Microsoft Licence Mobility Verification form](https://www.microsoft.com/en-us/licensing/licensing-programs/software-assurance-license-mobility.aspx) to use your own existing licensing
 
+- If you have the application licensing such as SQL on your own monthly reported SPLA. In this case, you will need to provide a confirmation to your UKCloud Service Delivery Manager or raise a support ticket and provide the SPLA details in case of an audit.
```
Comment (suehighmoor) [inline articles/shared/shared-how-license-microsoft.md:33]: Can you rephrase this so that it is more consistent with the other two options? E.g., something like:
You can use your own monthly reported SPLA, for example if you have SPLA application licensing for SQL. In this case...

**Review by vrush20 — APPROVED** [review]
Agree with Sue's feedback

**Review by suehighmoor — APPROVED** [review]
LGTM
